.changeset/flat-colts-learn.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/combobox': patch
+'@twilio-paste/core': patch
+---
+
+[Combobox] Fix internal refs so primitive doesn't override them
```

Comment:
You might need a more descriptive changelog entry for this as it's not refs related anymore